indexes/plugin.js

```diff
@@ -51,13 +52,15 @@ module.exports = function (
     } else cb()
   }
 
+  const liveWriteBatch = debounce(writeBatch, 250)
+
   function onData(record, isLive) {
-    // maybe check if for us!
     const unwritten = handleData(record, processed)
-    if (unwritten > 0) unWrittenOffset = record.offset
+    unWrittenOffset = record.offset
```

Comment:
Why was the if removed?

Reply:
That is this part: If an index doesn't write something for each message

Reply:
Ah I see